packages/app-transfer/src/Submit.tsx

```diff
@@ -18,7 +18,7 @@ type Props = I18nProps & {
   isDisabled: boolean,
   accountIndex?: BN,
   amount: BN,
-  from: Uint8Array,
+  from: string,
```

Comment:
Should `to` not also be accountId type? Not sure where it is converted now - e.g. transfer, the recipient gets selected from the InputAddress, passed through (previously it was U8a, now accountId). Basically my question is - does actual transactions still work? (Either via the extrinsics app or transfer?)